omnibus/files/private-chef-cookbooks/private-chef/templates/default/nginx/scripts/check_token_using_endpoint.lua.erb

```diff
@@ -0,0 +1,24 @@
+ngx.req.read_body()
+local res
+
+if ngx.var.request_method == "POST" then
+  meth = ngx.HTTP_POST
+else
+  meth = ngx.HTTP_GET
+end
+
+res = ngx.location.capture("/organizations/" .. ngx.var.request_org .. "/validate" .. ngx.var.request_uri,
+    { method = meth,
+      always_forward_body = true,
+      copy_all_vars = true,
```

Comment:
Scratch that comment about `copy_all_vars`, it's necessary (adam leff pointed out that this includes the original headers, of course we need those!). But `upstream` and `request_org` could still be unnecessary.